src/detect-transform-base64.c

```diff
@@ -149,15 +142,17 @@ static void TransformFromBase64Decode(InspectionBuffer *buffer, void *options)
         decode_length = nbytes;
     }
 
+    uint8_t decoded[input_len];
```

Comment:
Could we compute a more precise number ? (and smaller)

Reply:
yes. will do so.

Actually this was the reason I wanted to alloc a vector on the rust side so the entire base64 code including the calculation of exact decoded length could be done there. Now, it'll be making way in unrelated code like this.. ref https://github.com/OISF/suricata/pull/11544